src/PBot.Tests.Integration/Helper.cs

```diff
@@ -5,27 +5,35 @@
 //copy pasted from Octokit.net
 public static class Helper
 {
-    static readonly Lazy<Credentials> _credentialsThunk = new Lazy<Credentials>(() =>
```

Comment:
That's a different consideration:

1. make fields readonly
2. cater for those fields being of mutable types, e.g. project a dictionary into a new one in a property getter